--- app\src\Lud_Module.php
+++ PHP_CodeSniffer
@@ -25,11 +25,11 @@
 interface Lud_Module
 {
 
-	/**
-	 * Setups the module
-	 *
-	 * @return void
-	 */
+    /**
+     * Setups the module
+     *
+     * @return void
+     */
     public function setup();
 
 }
--- app\src\modules\Lud_Filter.php
+++ PHP_CodeSniffer
@@ -26,11 +26,11 @@
 
     private $_module_count = 0;
 
-	/**
-	 * Setups the module
-	 *
-	 * @return void
-	 */
+    /**
+     * Setups the module
+     *
+     * @return void
+     */
     public function setup() 
     {
         // TODO: Implement setup() method.

